src/resources/postcss/tickets-admin/settings/_tickets-commerce.pcss

```diff
@@ -463,3 +463,100 @@
 		color: #405bfb;
 	}
 }
+
+.tec-tickets__admin-settings-tickets-commerce-section-menu {
+	padding-top: var(--tec-spacer-6);
+}
+
+.tec-tickets__admin-settings-tickets-commerce-section-menu-link {
+	border-bottom-color: #c4c4c4;
+	border-bottom-style: solid;
+	border-bottom-width: 1px;
+	color: #23282d;
+	display: inline-block;
+	font-size: var(--tec-font-size-2);
+	margin-right: var(--tec-spacer-5);
+	padding-bottom: var(--tec-spacer-1);
+	text-decoration: none;
+}
+
+.tec-tickets__admin-settings-tickets-commerce-section-menu-link.active {
```

Comment:
Following BEM what we should do here is have a modifier class for the active, and then use both in the HTML.

The modifier class would be `.tec-tickets__admin-settings-tickets-commerce-section-menu-link--active` and then in the HTML `a` element you use both.